charts/hivemq-platform/values.yaml

```diff
@@ -110,7 +127,8 @@ license:
   additionalLicenses: {}
     ## Defines the name of the additional main HiveMQ license (the .lic suffix will be added automatically).
     #license-name:
-    #  # Inlines the additional main HiveMQ license as an encoded 64-bytes string.
+    #  # Inlines the additional main HiveMQ license as a string.
+    #  # By default, this data must be a Base64 encoded string. Otherwise, set isLicenseBase64Encoded=false to use a clear string.
```

Comment:
By default, this data is expected as Base64 encoded string. Set "isLicenseBase64Encoded=false" to provide a clear string.